noetic/distribution.yaml

```diff
@@ -5985,8 +5985,8 @@ repositories:
       - qb_move_hardware_interface
       tags:
         release: release/noetic/{package}/{version}
-      url: https://bitbucket.org/qbrobotics/qbmove-ros-release.git
-      version: 2.2.1-1
+      url: https://bitbucket.org/qbrobotics/qbdevice-ros-release.git
```

Comment:
This is failing CI because this is the wrong release repository for this section. Alternatively you're editing the wrong section. This is in the qb_move section not the qb_device section.